### ISSUE TITLE
pontine ischemic lacunes

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4951]
**Preferred term label:**
pontine ischemic lacunes

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lacunes are infarcts < 15 mm in diameter in the cortical white matter or in the corona radiata, internal capsule, centrum semiovale, thalamus, basal ganglia, or pons
https://www.ncbi.nlm.nih.gov/pubmed/16324092?dopt=Abstract

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0032325

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/618564

**Your nano-attribution (ORCID)**
